templates/settings/admin.php

```diff
@@ -55,6 +55,21 @@
 			<a id="save-min-confidence" class="icon-align icon-edit" style="display: none;" title="<?php p($l->t('Save'));?>" href="#"></a>
 		</p>
 		<br>
+		<h3>
+			<?php p($l->t('Minimum face size'));?>
+		</h3>
+		<p class="settings-hint"><?php p($l->t('Very small faces may be discovered during analysis, however they can be very diffuse to compare reliably. The smaller faces will not be clustered, but you will still be able to identify them.'));?>
```

Comment:
> but you will still be able to identify them.

What this means? If I set min face to 50x50px, they will not be in clustering, right? How can I "identify"them? If I open image with them, I will see them, but no person will be assigned to them?

Asking as this might be confusing to users (or maybe it is confusing to me only:D)

Reply:
> If I open image with them, I will see them, but no person will be assigned to them?

Now the face will now be clustered as a single photo person. 

> I will see them, but no person will be assigned to them?

This would be ideal. I always get annoyed by the `New Person ##` of the clusters.  But now it doesn't change.

I was thinking, make that the name admit NULL, and the user assigns his name directly.
* If the cluster has a name assigned, never re-cluster their faces.
* If the user assigns a NULL cluster to existing name,link to original cluster.
* So, re clustering just the faces on unnamed clusters.

But to do something like this, you have to have good clusters first, which is what I'm trying to improve these days.

Reply:
p.s: You can suggest the message.. :wink: